init.sql

```diff
@@ -1187,3 +1190,35 @@ EXECUTE FUNCTION trigger_fn_refresh_has_profile_picture_id();
 --------------------------------------------------------------------------------
 
 --------------------------------------------------------------------------------
+UPDATE banned_person
+SET email = CONCAT(
+    REPLACE(
+        SPLIT_PART(
+            SPLIT_PART(email, '@', 1),
+            '+', 1
+        ),
+        '.', ''
+    ),
+    '@gmail.com'
+)
+WHERE email LIKE '%@gmail.com' OR email LIKE '%@googlemail.com';
+
+ALTER TABLE person ADD COLUMN normalized_email TEXT;
```

Comment:
Something I forgot to mention was that these `ALTER TABLE` statements need to be idempotent, in case the migration only partially succeeds. So just add `IF NOT EXISTS` or whatever it is